### PR TITLE
Us contact messages

### PR DESCRIPTION
**Pull request by Marcel-Cathelin**
Add the page contact.

### CODE REVIEW

File: src/Controller/ContactController.php [hunk 1]
```diff
@@ -31,7 +31,8 @@ public function formulaire(): string
             }
 
             if (empty($errors)) {
-                $success = "Votre message a bien été pris en compte. Nous reviendrons vers vous dans les brefs délais.";
+                $success = "Merci pour votre message ! Il a bien été pris en compte.
+                 Nous vous recontacerons rapidement.";
```
Comment (aldaitzwild) [inline src/Controller/ContactController.php:35]: Typo : Nous vous recontacerons rapidement => Nous vous recontacterons rapidement

File: src/View/Login/form.html.twig [hunk 2]
```diff
@@ -18,6 +37,7 @@
 					{% endfor %}
 				{% endif %}
 			</div>
+>>>>>>> origin/dev
```
Comment (aldaitzwild) [inline src/View/Login/form.html.twig:40]: This is a conflict ! It shouldn't have been committed  ! Fix it

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by Marcel-Cathelin** [comment]
I made the changes as request.

File: Sauvegarde de ContactManager php.txt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+         $statement->execute();
+
+        return $this->pdo->lastInsertId();
+    }
+}
+
+
+ 
+$userId = $this->pdo->lastInsertId();
```
Comment (aldaitzwild) [inline Sauvegarde de ContactManager php.txt:36]: What is this file ??? You don't put backup file in a dev project !

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by Marcel-Cathelin** [comment]
Delete the .txt file.

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**